pkgs/development/python-modules/kdl-py/default.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "kdl-py";
+  version = "1.2.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-Y/P0bGJ33trc5E3PyUZyv25r8zMLkBIuATTCKFfimXM=";
+  };
+
+  build-system = [ setuptools ];
+
+  pythonImportsCheck = [ "kdl" ];
+
+  checkPhase = ''
+    runHook preCheck
+
+    python tests/run.py
+
+    runHook postCheck
+  '';
+
+  meta = {
+    description = "Parser for the KDL language";
+    homepage = "https://github.com/tabatkins/kdlpy";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ ];
```

Comment:
New packages must have a maintainer?

LGTM otherwise.

Reply:
Yes, they must.
But I am not familiar with or interested in those packages.
I will maintain them as a Python maintainer, but I don't want to be the maintainer of them because they make a lot of noise.😇